bundles/org.openhab.binding.mqtt.generic/src/main/resources/ESH-INF/config/color-channel-config.xml

```diff
@@ -11,6 +11,17 @@
 				received value is assigned to an item.</description>
 			<advanced>true</advanced>
 		</parameter-group>
+		
+        <parameter name="colorMode" type="text">
+            <label>Color Mode</label>
+            <description>Defines the color representation format of the payload. "HSB" by default.</description>
+            <options>
+                <option value="hsb">HSB (Hue, Saturation, Brightness)</option>
+                <option value="rgb">RGB (Red, Green, Blue)</option>
+                <option value="xyY">CIE xyY</option>
```

Comment:
```suggestion
                <option value="xyY">CIE xyY (x, y, Brightness)</option>
```
We should add the values for the xyY colorMode to the color-channel-config, so that people know which values they need for using this colorMode, since the third parameter is not the _real_ Y.

Reply:
@nwpr, have you seen any device with a pure CIE xyY implementation (with the real Y)? I'm not sure if we should call the channel "CIE xyY", because it is not a 100% accurate implementation. Do you know if there is any other name for the "(x, y, Brightness)" representation? It seems that xyB is not used :laughing: @J-N-K, what do you think about @nwpr's suggestion? Is that enough or should I rename all "xyY" references with another name? IMO, and knowing that the real "Y" is not even used in openHAB's core functions, we should be fine with ```CIE xyY (x, y, Brightness)``` :+1:

Reply:
No, but I understand your concerns. On one hand xyY doesn't reflect the real function of this mode, but on the other hand the official name for this color space is indeed CIE 1931 xyY.

I just searched for other implementations of this and found the following code in the Tradfri binding: [TradfriColor.java](https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.tradfri/src/main/java/org/openhab/binding/tradfri/internal/TradfriColor.java). They just call it xy in their code.

Reply:
Alright, you've convinced me! :wink: Name changed to ```CIE xyY (x, y, Brightness)```, now we simply have to wait for a maintainer's opinion.